typings/glamorous.test.tsx

```diff
@@ -0,0 +1,21 @@
+import glamorous from "../";
+
```

Comment:
This refers to Glamorous package. Typescript will look for package.json in that directory.